theme/less/components/tabs.less

```diff
@@ -9,26 +9,17 @@ Tabs allow for easy navigation between categories. You can toggle a `.active` cl
 Warning : `.tab` first immediate child (`.tab > *`) has a `pointer-events: none` property set so that clicks actually register to the tab element and not the children.
 This prevents messy event handling with bubbling prevention, but might cause you trouble in some edge cases.
 
-```btn-tabs.html
-<nav class="tabs">
-    <a class="btn-tab tab active" href="#">Selection Etalab</a>
-    <a class="btn-tab tab" href="#">Dernières réutilisations</a>
-    <a class="btn-tab tab" href="#">Voir plus</div>
+```tabs.html
```

Comment:
Do you think that we should keep  these `tabs` and `tab` in the future? Or could we remove those and use dsfr tabs or tags and move the logic left in tags if needed?

Reply:
We should remove this custom logic and rely on DSFR tabs only in the future to avoid usability and accessibility differences.

But in the meantime, I think it's best to keep them separate so we don't change the default behavior  of tags. Our current tab logic breaks the anchor typical flow.